vaadin-menu-bar-flow-parent/vaadin-menu-bar-flow/src/test/java/com/vaadin/flow/component/menubar/tests/MenuBarTest.java

```diff
@@ -100,4 +100,9 @@ private void assertChildrenAndItems(MenuItem... expected) {
         menuItems = menuBar.getItems().toArray();
         Assert.assertArrayEquals(expected, menuItems);
     }
+
+    @Test
+    public void close_smoke() {
+        menuBar.close();
```

Comment:
Needs an actual integration test. This test would pass even if the `close`-method was empty.

Reply:
I agree. Let me try to cobble up a TestBench test

Reply:
Added the `MenuBarPopupButtonIT` test which tests the close() function.